### DISCUSSION TITLE
Rename column functionality

### DISCUSSION THREAD

**Discussion by html5maker** [revolist/revogrid#476] [General]
Is there any way to add the "rename column" functionality to the revogrid?

**Comment by revolist** [comment]
Hello, not very clear what is rename column, but it is simple as just mutate RevoGrid.columns = [...columns] with new names, all things are sort of reactive.

**Comment by html5maker** [comment]
I mean to add this option to the UI (like ... dropdown menu with the rename option) for each column

**Comment by van-huyen** [comment]
In Vue2 (i18n nuxt app) to change columns name when language change: i fill col name in mouted() by i18n defined and update them like RevoGrid.columns = [...columns] 

if you user want to rename columns, save it in locationStore and update them like i18n
![image](https://user-images.githubusercontent.com/83264685/156953853-61d92d89-a0d8-4d7e-8131-0f608a43fa4e.png)

you can create context menu by right click (on grid - on header) by yourself:
![image](https://user-images.githubusercontent.com/83264685/156954150-a216d341-0c53-4f57-8348-b13282aeadaa.png)
